src/app/_models/depublication.ts

```diff
@@ -0,0 +1,33 @@
+export enum SortDirection {
+  UNSET = 'UNSET',
+  ASC = 'ASC',
+  DESC = 'DESC'
+}
+
+export class SortParameter {
+  field: string;
+  direction: SortDirection;
+}
+
+export enum DepublicationStatus {
+  PUBLISHED = 'PUBLISHED',
```

Comment:
I think this should be called `PENDING`. Just to avoid all sorts of confusions. Or `NOT_DEPUBLISHED` if you prefer. First of all, we don't know whether the record is published (it may not even exist). Secondly, the aim is to depublish, not to publish.